src/python/WMComponent/DBS3Buffer/MySQL/NewAlgo.py

```diff
@@ -11,19 +11,11 @@
 from WMCore.Database.DBFormatter import DBFormatter
 
 class NewAlgo(DBFormatter):
-    """
-    _NewAlgo_
 
-    Check for existence of algo, then add if not exists
-    """
-    existsSQL = """SELECT id FROM dbsbuffer_algo WHERE app_name = :app_name AND
```

Comment:
This code will break if we try to insert a set of values that already exist in the database. Any reason to remove this check?